### PR TITLE
EVG-6528: explicitly set host when installing Jasper service

### PR DESCRIPTION
**Pull request by Kimchelly**
JIRA: https://jira.mongodb.org/browse/EVG-6528

### CODE REVIEW

File: model/host/hostutil.go [hunk 1]
```diff
@@ -238,7 +238,7 @@ func (h *Host) FetchAndReinstallJasperCommand(config evergreen.HostJasperConfig)
 // delete the current Jasper service configuration (if it exists), install the
 // new configuration, and restart the service.
 func (h *Host) ForceReinstallJasperCommand(config evergreen.HostJasperConfig) string {
-	params := []string{fmt.Sprintf("--port=%d", config.Port)}
+	params := []string{"--host=0.0.0.0", fmt.Sprintf("--port=%d", config.Port)}
```
Comment (bsamek) [inline model/host/hostutil.go]: What does it mean to set host to 0.0.0.0 for jasper?

Comment (Kimchelly) [reply model/host/hostutil.go]: If the host isn't specified, Jasper will listen on localhost by default so it's only reachable from connections from the same host. Using 0.0.0.0 allows it to listen on all available IPs for that host, which lets the app server connect to Jasper remotely using RPC.

Comment (bsamek) [reply model/host/hostutil.go]: Doesn't that mean that it will listen potentially listen on ports that are open to the world?

Comment (Kimchelly) [reply model/host/hostutil.go]: As far as I know, the firewall rules only allow traffic from the evergreen app servers in the port range 1024-2500.

Comment (bsamek) [reply model/host/hostutil.go]: Sorry, totally missed that this applies to IP addresses, not to ports.

**Pull request merged**